engine/src/main/java/nl/inl/blacklab/search/results/Hits.java

```diff
@@ -691,7 +696,9 @@ public Hits sample(SampleParameters sampleParameters) {
             }
         });
 
-        return Hits.fromList(queryInfo(), sample, null, sampleParameters, sample.size(), docsInSample.getValue(), docsInSample.getValue(), capturedGroups);
+        return Hits.fromList(queryInfo(), sample, null, sampleParameters, sample.size(),
+                docsInSample.getValue(), docsInSample.getValue(), capturedGroups,
+                false);
```

Comment:
Replace `false` with `hasAscendingLuceneDocIds()`? chosenHitIndices uses a TreeSet so should be sorted right, so the sample doesn't reorder the hits?